internal/configtxgen/encoder/encoder.go

```diff
@@ -672,3 +678,35 @@ func (bs *Bootstrapper) GenesisBlock() *cb.Block {
 func (bs *Bootstrapper) GenesisBlockForChannel(channelID string) *cb.Block {
 	return genesis.NewFactoryImpl(bs.channelGroup).Block(channelID)
 }
+
+func addBFTBlockPolicy(consenterProtos []*cb.Consenter, ordererGroup *cb.ConfigGroup) {
+	n := len(consenterProtos)
+	f := (n - 1) / 3
+
+	var identities []*mspa.MSPPrincipal
+	var pols []*cb.SignaturePolicy
+	for i, consenter := range consenterProtos {
+		pols = append(pols, &cb.SignaturePolicy{
+			Type: &cb.SignaturePolicy_SignedBy{
+				SignedBy: int32(i),
```

Comment:
index is added as the node/consenter id, index starts from 0 whereas the node id/consenter's id starts from 1. Should be increment by 1?

Reply:
it should start from 0